development/bootstrap-job-preparation/bootstrap-job.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+if [ $# -ne 3 ]; then
+  echo "Usage: $0 <bootstrap_job_version> <repo_username> <repo_api_key>"
+  exit 1
+fi
+
+BOOTSTRAP_JOB_VERSION=$1
+export ARTIFACTS_CREDS_USR=$2
+export ARTIFACTS_CREDS_PSW=$3
+
+mvn clean process-resources -Dbootstrap.job.version="${BOOTSTRAP_JOB_VERSION}" -s settings.xml
```

Comment:
Why do we need custom `settings.xml`? Why can't we rely on existing configuration? One of the requirements for local development is [Maven Configuration](https://backbase.io/documentation/backend-devkit/18.0.1/getting-started/configure-maven)

Reply:
Well that's true. but assuming everyone following the standard will be too optimistic. I prefer to make LBS self sufficient as much as possible.

Reply:
It should be a prerequisite for using LBS. Assuming developer has Java, Docker, etc on their machine is also too optimistic?

Reply:
those are tools not configuration. anyway, do you thing it is harmful have a working maven settings working for BSJ within LBS?

Reply:
Ok I will stop continuing this discussion. Just removed the setting. Again, it happened to me several times developers came to me complaining about the build failure. In the end it appeared that they are using a customised setting only compatible with their respective services not the standard one. We will continue supporting them.